src/share/dateRangePicker/reducer.ts

```diff
@@ -0,0 +1,50 @@
+import {
```

Comment:
I don't think that reducer for shared date range picker it's a good idea. Shared components should not be tightly coupled with your infra (state management library and etc.). Shared components should be flexible so that you could use them using redux, mobX, bare react with components state and etc.
This should be fixed